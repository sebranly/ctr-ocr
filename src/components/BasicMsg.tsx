import * as React from 'react';

export interface BasicMsgProps {
  msg: string;
}

const BasicMsg: React.FC<BasicMsgProps> = (props) => {
  const { msg } = props;

  return <div className="text-center mb">{msg}</div>;
};

export { BasicMsg };
