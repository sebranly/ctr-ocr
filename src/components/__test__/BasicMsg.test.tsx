import * as React from 'react';
import { render } from '@testing-library/react';

import { BasicMsg } from '../BasicMsg';

test('BasicMsg', () => {
  const createComponent = () => <BasicMsg msg="bonjour" />;

  const { container } = render(createComponent());
  expect(container.childNodes).toMatchSnapshot();
});
