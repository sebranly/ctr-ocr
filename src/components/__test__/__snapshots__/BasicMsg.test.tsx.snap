// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BasicMsg 1`] = `
NodeList [
  <div
    class="text-center mb"
  >
    bonjour
  </div>,
]
`;
